routes/api/landing.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const router = express.Router();
+
+const City = require("../../models/city");
+
+router.get("/", function (req, res) {
+  City.find({}, function (err, doc) {
+    if (err) { 
+      res.status(400).json(err); 
+    }
+    var destinations = [];
+    var i;
+    //change loop limit if want more featured cities
+    var index = Math.floor(Math.random() * doc.length)
+    var headImg = doc[index].imgMain
+    var headCityName = doc[index].name
+    for (i = 0; i < 5; i++) {
```

Comment:
are we going to have more than 4 cities?

Reply:
sometimes we show 3, sometimes 4, sometimes 5 pictures under "Featured Cities"

Reply:
Will be done in different story.